Source/WebCore/page/EventHandler.cpp

```diff
@@ -3066,6 +3075,19 @@ void EventHandler::clearElementUnderMouse()
     imageOverlayController->elementUnderMouseDidChange(protectedFrame(), nullptr);
 }
 
+bool EventHandler::isElementAnAncestorOfLastElementUnderMouse(Element* element) const
+{
+    if (!element)
+        return false;
+
+    for (WeakPtr weakElement : m_ancestorsOfLastElementUnderMouse) {
+        if (weakElement.get() == element)
```

Comment:
This already exists:

```cpp
inline bool operator==(const WeakPtr<T, WeakPtrImpl, PtrTraits>& a, U* b)
```

so you can:
```suggestion
        if (weakElement == element)
```